CRM/Civicase/Form/Report/BaseExtendedReport.php

```diff
@@ -59,6 +87,37 @@ protected function getFiltersTemplateName() {
     return 'Filters';
   }
 
+  /**
+   * Returns Custom fields meta data.
+   *
+   * @param array $customFields
+   *   Custom fields.
+   *
+   * @return array
+   *   Custom fields meta data.
+   */
+  private function getCustomFieldsMeta($customFields) {
+    $optionGroupIds = [];
+    $sortedLists = [];
+    foreach ($customFields as $customField) {
+      if (!empty($customField['option_group_id'])) {
+        $optionGroupIds[] = $customField['option_group_id'];
+      }
+    }
+    $extendsString = implode("','", $optionGroupIds);
+    $ogDAO = CRM_Core_DAO::executeQuery("
+    SELECT ov.value, ov.label, ov.option_group_id
+    FROM civicrm_option_value ov
+    WHERE ov.option_group_id IN ('" . $extendsString . "')
+    ORDER BY ov.weight;");
```

Comment:
@tunbola  Can you check if the code format is correct on excecuteQuery function?